scripts/update_python_version.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+"""Update the ITKPythonPackage version based off the ITK Git nightly-master
+branch."""
+
+import argparse
+import sys
+import os
+import subprocess
+
+argparser = argparse.ArgumentParser(description=__doc__)
+argparser.add_argument('itkSourceDir')
+
+args = argparser.parse_args()
+itkSourceDir = args.itkSourceDir
+if not os.path.exists(os.path.join(itkSourceDir, '.git')):
+    print('itkSourceDir does not appear to be a git repository!')
+    sys.exit(1)
+
+itkPythonPackageDir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+os.chdir(itkSourceDir)
+# "2017-02-08 15:21:09 -0500"
+commitDate = subprocess.check_output(['git',
+    'show', '-s', '--date=local', '--format="%ci"'])
```

Comment:
The correct syntax is:

```
git show -s --format="%cd" --date=local
```

In that case, the suffix `-0500` doesn't show up as expected